index.html

```diff
@@ -1066,25 +1141,86 @@ <h4>Listing</h4>
-                        </span>
+
+                        <p>
+                            The following example provides a walk-through of the paginated
+                            retrieval of TDs:
+                            
+                            <aside class="example" id="example-pagination"
+                                title="Query 12 TDs with page size 10.">
+                                The client requests the list and the total count of TDs, with a `limit` of 10:
+                                <pre>
+                                    GET /td?limit=10&count=true HTTP/1.1   
+                                    Host: tdd.example.com
+                                </pre>
+                                Response:
+                                <pre>
+                                    HTTP/1.1 200 OK
+                                    Content-Range: TDs 0-9/12
+                                    Content-Type: application/ld+json
+                                    Link: &#x3C;https://tdd.example.com/td?offset=10&#x26;limit=10&#x3E;; rel=&#x22;next&#x22;
+                                    Link: &#x3C;https://tdd.example.com/td&#x3E;; rel=&#x22;canonical&#x22;; etag=&#x22;842542a4-abf4-4fca-a38d-ff48f6dc3088&#x22;
+                                    Link: &#x3C;http://www.w3.org/ns/td#Thing&#x3E;; rel=&#x22;type&#x22;,
+                                        &#x3C;http://www.w3.org/ns/ldp#Page&#x3E;; rel=&#x22;type&#x22;
+
+                                    [{ TD }, ...9 other TDs... ]
+                                </pre>
+                                The server response has a Content-Range header with range values and
+                                total count of TDs. The Content-Type indicates that the response is JSON-LD. 
+                                There are several Links: `next` Link to query the remaining TDs,
+                                `canonical` Link pointing to the collection endpoint with `etag` value set
+                                to a UUID representing the current state of the collection
+                                state, and two other Links for `type` relations. 
+                                The body is an array of 10 TDs.
+                                
+                                <br><br>
+                                The client requests the next page, either by programmatically
+                                constructing the URL or by using the `next` Link from the previous response:
+                                <pre>
+                                    GET /td?offset=10&limit=10 HTTP/1.1   
+                                    Host: tdd.example.com
+                                </pre>
+
+                                Response:
+                                <pre>
+                                    HTTP/1.1 200 OK
+                                    Content-Range: TDs 10-12/*
```

Comment:
I believe the range should be "10-11"
Otherwise it would be 3 more TDs and the count 12 does not match (0-11)